core/src/main/java/tc/oc/pgm/kits/KitParser.java

```diff
@@ -435,6 +440,28 @@ private List<Color> parseColors(List<Node> nodes) throws InvalidXMLException {
     return colors;
   }
 
+  public ItemStack parseBanner(Element el) throws InvalidXMLException {
+    ItemStack itemStack = parseItem(el, Materials.BANNER);
+    BannerMeta meta = (BannerMeta) itemStack.getItemMeta();
+    DyeColor color = XMLUtils.parseDyeColor(XMLUtils.getRequiredAttribute(el, "base-color"));
+    List<org.bukkit.block.banner.Pattern> patterns = new ArrayList<>();
+    patterns.add(new org.bukkit.block.banner.Pattern(color, PatternType.BASE));
+    for (Element elLayer : el.getChildren("layer")) {
+      DyeColor layerColor = XMLUtils.parseDyeColor(XMLUtils.getRequiredAttribute(elLayer, "color"));
+      String patternString = XMLUtils.getRequiredAttribute(elLayer, "pattern")
+          .getValue()
+          .toUpperCase()
+          .replace(" ", "_");
+      PatternType patternType = PatternType.valueOf(patternString);
```

Comment:
this is not how you parse enums in pgm, use the utils, at the least:
```suggestion
      var patternType = XMLUtils.parseEnum(XMLUtils.getRequiredAttribute(elLayer, "pattern"), PatternType.class);
```
and if you're feeling fancy you can look into the XmlFluentParser which creates, imo, nicer code